.golangci.yaml

```diff
@@ -15,11 +15,18 @@ issues:
 linters:
   disable-all: true
   enable: # please keep this alphabetized
+    - gocritic
     - ineffassign
     - staticcheck
     - unused
 
 linters-settings: # please keep this alphabetized
+  gocritic:
+    enabled-checks:
+      - ruleguard
+    settings:
+      ruleguard:
+        rules: "ruleguard_rules.go"
```

Comment:
`hack/ruleguard/rules.go` ???

Reply:
The path needs to be the same in every module :(  Gross, but that is how ruleguard works

Reply:
Ehh, MAYBE I could do this in the root module.  I can try...